spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/ribbon/DefaultServerIntrospector.java

```diff
@@ -30,8 +30,12 @@
  */
 public class DefaultServerIntrospector implements ServerIntrospector {
 
-	@Autowired
-	private ServerIntrospectorProperties serverIntrospectorProperties;
+	private ServerIntrospectorProperties serverIntrospectorProperties = new ServerIntrospectorProperties();
+
+	@Autowired(required = false)
```

Comment:
Yes the default is the initialized field. If auto confoguration is enabled then the `ServerIntrospectorProperties` will set via autowired.

This are the Test which use the `DefaultServerIntrospector`  class:

spring-cloud-netflix-core\src\test\java\org\springframework\cloud\netflix\feign\encoding\FeignAcceptEncodingTests.java
spring-cloud-netflix-core\src\test\java\org\springframework\cloud\netflix\feign\encoding\FeignContentEncodingTest.java
spring-cloud-netflix-core\src\test\java\org\springframework\cloud\netflix\feign\ribbon\FeignLoadBalancerTests.java
spring-cloud-netflix-core\src\test\java\org\springframework\cloud\netflix\feign\ribbon\FeignRibbonClientTests.java
spring-cloud-netflix-core\src\test\java\org\springframework\cloud\netflix\feign\ribbon\RetryableFeignLoadBalancerTest.java
spring-cloud-netflix-core\src\test\java\org\springframework\cloud\netflix\ribbon\RibbonClientConfigurationTests.java
spring-cloud-netflix-core\src\test\java\org\springframework\cloud\netflix\ribbon\RibbonLoadBalancedRetryPolicyFactoryTest.java
spring-cloud-netflix-core\src\test\java\org\springframework\cloud\netflix\ribbon\RibbonLoadBalancerClientTests.java
spring-cloud-netflix-core\src\test\java\org\springframework\cloud\netflix\zuul\filters\route\okhttp\OkHttpRibbonCommandIntegrationTests.java